zad3/app/views/layouts/application.html.erb

```diff
@@ -2,14 +2,15 @@
 <html>
   <head>
     <title>Zad3</title>
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
+    <!%= csrf_meta_tags %>
+    <!%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <!%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```

Comment:
`!` spowodowały, że uciekły style - przynajmniej u mnie 😉 
![screen shot 2018-11-20 at 20 09 16](https://user-images.githubusercontent.com/18199403/48796581-4d738780-ed00-11e8-91ea-49ebd8742e92.png)

Reply:
właśnie kurde próbowałem coś z tym zrobić ale na tej linijce zawsze się wykrzaczało, więc chcąc zdążyć przed północą to wykomentowałem, żeby przynajmniej się kompilowało ;)